src/extension.ts

```diff
@@ -20,4 +21,9 @@ export function activate(context: vscode.ExtensionContext) {
 	return commonActivate(context, view);
 }
 
-export function deactivate() { }
+export async function deactivate() {
+	// Do our best to log out of all sessions
```

Comment:
does this need to be added to web-extension.ts as well?

Reply:
Good catch. Now done.